.github/ISSUE_TEMPLATE/graduation.md

```diff
@@ -0,0 +1,34 @@
+# Application for Graduation
+- [ ] Have committers from at least two organizations.
```

Comment:
Nit: "Committer" can have multiple meanings. Is this clearly defined in this context? Alternatively, what about "full members with voting rights" or so? Also, should it refer to "companies" instead of "organizations"?

Reply:
Defined earlier in the README